modules/ipi-install-troubleshooting-bootstrap-vm-cannot-boot.adoc

```diff
@@ -32,7 +31,7 @@ $ ssh core@172.22.0.2
 [core@localhost ~]$ sudo podman logs -f <container-name>
 ----
 +
-Replace `<container-name>` with one of `ironic-api`, `ironic-conductor`, or `ironic-inspector`. If you encounter an issue where the control plane nodes are not booting up via PXE, check the `ironic-conductor` pod. The `ironic-conductor` pod contains the most detail about the attempt to boot the cluster nodes, because it attempts to log in to the node over IPMI.
+Replace `<container-name>` with one of `ironic` or `ironic-inspector`. If you encounter an issue where the control plane nodes are not booting up from PXE, check the `ironic` pod. The `ironic` pod contains the most detail about the attempt to boot the cluster nodes, because it attempts to log in to the node over IPMI.
```

Comment:
One nit: 

s/ detail / details. Right? Wouldn't it be plural? I see that it wasn't previously. Alternatively, it could be like "the most detailed overview. . ." (which I like more). Feel free to ignore if I'm incorrect, as the context could determine this.

Reply:
Hi @stevsmit, I [revised](https://github.com/openshift/openshift-docs/pull/57027/files#diff-9334faadf96539f60825900554c3bb2b290ecbbb29edfb9ebcb2d620373053faR34) the text to drop the problematic bit. Can you PTAL? Thank you!